src/backend/columnar/README.md

```diff
@@ -0,0 +1,373 @@
+cstore_fdw
```

Comment:
we might forget about this file, could maybe omit it / its contents.

Reply:
makes sense, will see what is in there and remove the file if nothing of importance is there.

I think it mostly contains the fdw api. Given we are removing that anyway in a couple of weeks I think omitting this file is ok..